.github/workflows/close-stale.yml

```diff
@@ -0,0 +1,35 @@
+name: Mark stale issues and pull requests
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - codeql/.github/workflows/close-stale.yml
+  schedule:
+    - cron: "30 1 * * *"
+
+jobs:
+  stale:
```

Comment:
You may want to add `if: ${{ github.repository == 'github/codeql }}`  so that this doesn't run on forks.

Reply:
Done